include/ccf/service/tables/members.h

```diff
@@ -21,6 +21,22 @@ namespace ccf
   DECLARE_JSON_ENUM(
     MemberStatus,
     {{MemberStatus::ACCEPTED, "Accepted"}, {MemberStatus::ACTIVE, "Active"}});
+
+  enum class MemberRecoveryRole
+  {
+    NonParticipant = 0,
+    Participant,
+
+    /** If set then the member is to receive the full wrapper key
```

Comment:
```suggestion
    /** If set then the member is to receive a key
```

Right now it is true that it is the full wrapper key, but that could change without affecting the functionality, so I think it's best to describe this from the user perspective.

Reply:
Taken